### PR TITLE
Update config-settings.rst for AutoFollowThreads

### PR DESCRIPTION
**Pull request by esethna**

### CODE REVIEW

File: source/administration/config-settings.rst [hunk 1]
```diff
@@ -4201,6 +4201,20 @@ There are a number of settings customizable in ``config.json`` unavailable in th
 Service Settings
 ~~~~~~~~~~~~~~~~
 
+Automatically Follow Threads
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+This setting has been added as a requirement to support [Collapsed Reply Threads](https://docs.google.com/presentation/d/1QSrPws3N8AMSjVyOKp15FKT7O0fGMSx8YidjSDS4Wng/edit#slide=id.g2f0aecc189_0_245) (releasing in beta in Q1 2021) and may affect server performance. It is recommended to review our [documentation on hardware requirements](https://docs.mattermost.com/install/requirements.html#hardware-requirements) to ensure your servers are appropriately scaled for the size of your user base.
```
Comment (amyblais) [inline source/administration/config-settings.rst:4207]: Link formatting is broken here

File: source/administration/config-settings.rst [hunk 2]
```diff
@@ -4201,6 +4201,20 @@ There are a number of settings customizable in ``config.json`` unavailable in th
...
+
+**True**: Threads a user starts, participates in or is mentioned in are automatically followed. A new ``threads`` table is added in the database that tracks followed threads for each user, in addition to read or unread state of each followed thread.
```
Comment (amyblais) [inline source/administration/config-settings.rst:4209]:
```suggestion
**True**: Threads a user starts, participates in, or is mentioned in are automatically followed. A new ``threads`` table is added in the database that tracks followed threads for each user, in addition to read or unread state of each followed thread.   
```

File: source/administration/config-settings.rst (hunk 1 above)
Comment (esethna) [reply source/administration/config-settings.rst]: fixed @amyblais

**Review by amyblais — APPROVED** [review]

File: source/administration/config-settings.rst [hunk 3]
```diff
@@ -4201,6 +4201,20 @@ There are a number of settings customizable in ``config.json`` unavailable in th
 Service Settings
 ~~~~~~~~~~~~~~~~
 
+Automatically Follow Threads
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+This setting has been added as a requirement to support `Collapsed Reply Threads <https://docs.google.com/presentation/d/1QSrPws3N8AMSjVyOKp15FKT7O0fGMSx8YidjSDS4Wng/edit#slide=id.g2f0aecc189_0_245>`_ (releasing in beta in Q1 2021) and may affect server performance. It is recommended to review our `documentation on hardware requirements <https://docs.mattermost.com/install/requirements.html#hardware-requirements>`_ to ensure your servers are appropriately scaled for the size of your user base.
+
+**True**: Threads a user starts, participates in, or is mentioned in are automatically followed. A new ``threads`` table is added in the database that tracks followed threads for each user, in addition to read or unread state of each followed thread.
```
Comment (reflog) [inline source/administration/config-settings.rst:4209]: `Threads` table only lists the threads and their participants. `ThreadMembership` table tracks what threads does a user follow and the read state.

**Review by reflog — CHANGES_REQUESTED** [review]

File: source/administration/config-settings.rst (hunk 3 above)
Comment (esethna) [reply source/administration/config-settings.rst]: Updated

**Review by reflog — APPROVED** [review]

**Pull request merged**